app/controllers/newflow/educator_signup_controller.rb

```diff
@@ -33,6 +33,7 @@ def educator_signup
 
     def educator_change_signup_email_form
       @email = unverified_user.email_addresses.first.value
+      @total_steps = 4
```

Comment:
personally, i really dislike "magic numbers" (better to put all this state stuff into a PORO that's passed as a component of a view-controller interface adapter object.  but that would be a lot more work.

Reply:
Would it be worse or better if there was no instance variable for the number 4. I feel like I went one (small) step beyond just putting it in the view—by putting it as an instance variable. We're still (as a team) not totally decided if we are ever going to skip the entire step 3 or... not, that's still in the works but yeah, for now I think that it doesn't hurt to keep this instance variable—for now.

Reply:
Again, for the record, I also dislike "magic numbers" and I agree about creating a PORO (a ModelView object), but those are small improvements I hope/plan to do soon in a future PR.